nle/dataset/db.py

```diff
@@ -228,7 +229,7 @@ def create(filename=DB):
             (
                 gameid        INTEGER,
                 dataset_name  TEXT,
-                PRIMARY KEY (gameid, dataset_name)
+                PRIMARY KEY   (dataset_name, gameid)
```

Comment:
Out of curiosity, why the swap from (gameid, dataset_name)?

Reply:
for multicolumn indices the order is important. sqlite creates  sorts the table on the first value, and then sorts duplicated values by the second column. in this case (clearly) in most cases, the dataset_name is what we are searching for, then maybe gameid.